README.rst

```diff
@@ -1,4 +1,4 @@
-.. image:: https://github.com/pymc-devs/pymc3/blob/master/docs/pymc3_logo.jpg?raw=true
+.. image:: https://cloud.githubusercontent.com/assets/781659/26266864/0d462598-3c9c-11e7-8697-2e811e231694.png
```

Comment:
oh, this is why i suggested two PR's -- the second one will change this back (it will not render with `/blob/master/`  until it is merged!)